db/migrate/20221212192830_create_itt_providers.rb

```diff
@@ -0,0 +1,15 @@
+class CreateIttProviders < ActiveRecord::Migration[6.1]
+  def change
+    create_table :itt_providers do |t|
+      t.text :legal_name, unique: true
+      t.text :operating_name
+      t.date :removed
+      t.date :added
```

Comment:
Do we need added when we already have created_at?

Reply:
possibly... i did think this then thought of reason why not i can have another look